src/main/java/com/example/assignment/EmployeeController.java

```diff
@@ -12,10 +19,39 @@
 	@Autowired
 	private EmployeeService employeeService;
 	
-	@PostMapping(path="/create")
+	@PostMapping("/create")
```

Comment:
When you create endpoints related to same resource, you should have the noun as the mapping with GET and POST differentiating what they do. For eg if we create employees, the requests should be

/employee (POST) to create
/employee/{id} (PUT) to update
/employee/{id} (GET) to get an employee
/employees (GET) list all employees
/employee/{id} (DELETE) delete an employee